engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/RecordScreen.java

```diff
@@ -107,29 +104,16 @@ protected void initWidgets() {
         close = find("close", UIButton.class);
     }
 
-    private void loadGame(GameInfo item) {
-        try {
-            final GameManifest manifest = item.getManifest();
-            copySaveDirectoryToRecordingLibrary(manifest.getTitle());
-            recordAndReplayUtils.setGameTitle(manifest.getTitle());
-            config.getWorldGeneration().setDefaultSeed(manifest.getSeed());
-            config.getWorldGeneration().setWorldTitle(manifest.getTitle());
-            CoreRegistry.get(GameEngine.class).changeState(new StateLoading(manifest, NetworkMode.NONE));
-        } catch (Exception e) {
-            logger.error("Failed to load saved game", e);
-            getManager().pushScreen(MessagePopup.ASSET_URI, MessagePopup.class).setMessage("Error Loading Game", e.getMessage());
-        }
-    }
-
-    private void copySaveDirectoryToRecordingLibrary(String gameTitle) {
-        File saveDirectory = new File(PathManager.getInstance().getSavePath(gameTitle).toString());
-        Path destinationPath = PathManager.getInstance().getRecordingPath(gameTitle);
-        File destDirectory = new File(destinationPath.toString());
-        try {
-            FileUtils.copyDirectoryStructure(saveDirectory, destDirectory);
-        } catch (Exception e) {
-            logger.error("Error trying to copy the save directory:", e);
-        }
+    /**
+     * Launches {@link NameRecordingScreen} with the info of the game selected in this screen.
+     *
+     * @param nameRecordingScreen The instance of the screen to launch
+     * @param info The info of the selected game.
+     */
+    private void launchRename(NameRecordingScreen nameRecordingScreen, GameInfo info) {
```

Comment:
Since you are not really renaming something you might want to change this to something like "nameRecording"

Reply:
I have renamed this function to `launchNamingScreen`.